tests/integration/abort-while-waiting-for-trigger.cpp

```diff
@@ -0,0 +1,78 @@
+#include "acquire.h"
```

Comment:
It's pretty obvious what this test is for, but I like that we've established a precedent for docstrings on all new tests. We should stick to it.